### DISCUSSION TITLE
Set default branch to gh-pages

### DISCUSSION THREAD

**Discussion by nickserv** [JamesIves/github-pages-deploy-action#1151] [Ideas]
GitHub Pages will automatically be enabled if you push a `gh-pages` branch to a repository, so I think this action should be able to just do that by default to make configuration simpler. The similar [peaceiris/actions-gh-pages](https://github.com/peaceiris/actions-gh-pages) action already does this.

**Comment by nickserv** [comment]
Would a PR be helpful or do we want to reach a consensus first?

**Comment by JamesIves** [comment]
I think this would be a useful PR if you'd like to make it!

**Comment by nickserv** [comment]
This is done.
